mmtrack/datasets/sot_coco_dataset.py

```diff
@@ -22,7 +23,13 @@ def __init__(self, ann_file, *args, **kwargs):
             ann_file (str): The official coco annotation file. It will be
                 loaded and parsed in the `self.load_data_infos` function.
         """
-        self.coco = COCO(ann_file)
+        if 'file_client_args' in kwargs and kwargs['file_client_args'][
+                'backend'] != 'disk':
+            self.file_client = mmcv.FileClient(**kwargs['file_client_args'])
+            with self.file_client.get_local_path(ann_file) as local_path:
+                self.coco = COCO(local_path)
+        else:
+            self.coco = COCO(ann_file)
         super().__init__(*args, **kwargs)
```

Comment:
Too ugly here, we may only keep line 28-30?

Reply:
If only keep line 28-30, it will create a temporary file and read it even reading local files.

Reply:
I think it is acceptable. And mmdet also use [the same way](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/datasets/custom.py#L93-L95)

Reply:
OK， I modify it right now.